ClientTests/DefaultSearchPrefsTests.swift

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+@testable import Client
+import UIKit
+
+import XCTest
+
+class DefaultSearchPrefsTests: XCTestCase {
+
+    func testParsing() {
+        // setup the list json
+        let searchPrefs = DefaultSearchPrefs(with: Bundle.main.resourceURL!.appendingPathComponent("SearchPlugins").appendingPathComponent("list.json"))!
+
+        // setup the most popular locales
```

Comment:
How do we know we're not shipping broken XML files to Indonesia or India?

For that matter, how do we know we're not shipping broken `list.json`? (smoke testing aside)

Reply:
The good thing about this setup vs the old one is. The XML files are all shared between countries. So if we test the global xml file in a test for en_CA then the XML file should also work in India.

As for List.json, Hopefully we arent merging patches to master without CI 😄